README.md

```diff
@@ -74,12 +75,46 @@ Unlike `npm`, `yarn` doesn't require the `run` command. So if you want to run `l
 yarn lint
 ```
 
+### How to contribute?
+
+- Clone the project locally from GitHub - `git clone https://github.com/sebgroup/green.git`.
```

Comment:
We should probably add the fork option as well for users who don't belong to our github.com organisation. It's basically the same but they need to fork it first:)